p/themes/base-theme/template.css

```diff
@@ -949,6 +949,59 @@ a.website:hover .favicon {
 	max-width: 550px;
 }
 
+.content header {
+	color: #666;
+	font-size: 90%;
```

Comment:
There could also be a `<header>` element inside the content, so I'd rather see a rem value.

Reply:
thanks. fixed it.

Reply:
Hm, I'm not convinced. :-) I'd rather see a rem value, because em or percent is almost never what you actually want and then you don't have to "fix" anything.

Reply:
I'd be inclined to say that for `font-size`, a unit like em or percent simply shouldn't ever be used at all, since it's after January 31, 2020 (IE10 EOL).

Reply:
I agree with you. We need to fix it in every theme. There is a mix of `rem`, `em`, `%`

Reply:
fixed it